### PR TITLE
Optimize documents and synchronize English versions

### PR DESCRIPTION
**Pull request by samzong**
### Ⅰ. Describe what this PR does

- [x] Optimize documents
- [x] synchronize English versions

### Ⅱ. Does this pull request fix one issue?

fixes #XXXX

### Ⅴ. Special notes for reviews

This is my first PR, If there are any inaccuracies, I hope you can correct me.

### CODE REVIEW

File: docs/en/userguide/faq.md [hunk 1]
```diff
@@ -132,24 +138,24 @@ Change: 2021-01-19 00:16:21.539559082 +0800
...
-First, you need to use the command `kubectl get csidriver` to check whether the CSI driver is installed.
+First, you must use the command `kubectl get csidriver` to check whether the CSI driver is installed.
 If not, you should use the command `kubectl apply -f charts/fluid/fluid/templates/CSI/driver.yaml` to install it, and then observe whether the PVC is successfully mounted into the application.
 If it is not solved, you shall use the command `export KUBECONFIG=/etc/kubernetes/kubelet.conf && kubectl get csidriver` to check Kubelet whether has permission to see the CSI Driver or not. 
 
-## 8. After creating Dataset and AlluxioRuntime，When does alluxio master Pod enter into CrashLoopBackOff state？
+## 8. After creating Dataset and AlluxioRuntime， When does alluxio master Pod enter into the CrashLoopBackOff state？
 
-**Answer**:First, use command`kubectl describe pod <dataset_name>-master-0 `to query the reason why did Pod exit by mistake.
+**Answer**:First, use the command kubectl describe pod <dataset_name>-master-0 `to query the reason why did Pod exit by mistake.
```
Comment (cheyang) [inline docs/en/userguide/faq.md:152]: I think you also need to keep "`" between word command and kubectl.

Comment (samzong) [reply docs/en/userguide/faq.md:152]: great!

Comment (samzong) [reply docs/en/userguide/faq.md]: Thanks for your review @cheyang, I've fixed and optimized some others.

looking forward to your feedback

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**